### PR TITLE
Glaph

### PR DESCRIPTION
**Pull request by hiroaki-iwase**
I developed connection glaph by use dyglaph and graphael.

### CODE REVIEW

File: app/helpers/connection_helper.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    target_log = {}
+    hash.each{|instance, logs|
+      target_log[instance] = {}
+      logs.each{|log|
+        if log =~ /^I, \[([-\d]+)T([\d:]+)\.\d+\s#\d+\]\s+INFO -- : Connected from [\d.:]+\sI have (\d+) connections.$/
+          target_log[instance].store("#{$1} #{$2}", $3)
+        end
+      }
+    }
+    m_array = merge_connection_count(target_log)
```
Comment (banrui) [inline app/helpers/connection_helper.rb]: What's "m_array"?? If you have other idea of variable name, please modify it more clearly.

File: app/assets/javascripts/connection.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                    strokeWidth: 3,
+                    strokeBorderWidth: 1,
+                    highlightCircleSize: 5
+                },
+                connectSeparatedPoints: true,
+                drawPoints: true
+            }
+        );
+
+        var makeClickCallback = function(graph) {
```
Comment (banrui) [inline app/assets/javascripts/connection.js]: Maybe, we do not need this var.

Comment (hiroaki-iwase) [inline app/assets/javascripts/connection.js]: Yes!
I will remove it.

File: app/helpers/connection_helper.rb (hunk 1 above)
Comment (hiroaki-iwase) [inline app/helpers/connection_helper.rb]: m_array means "merged_connection_count_array".
but I agree it is ambiguous.
so I changed to "conn_cnt_array".

**Comment by hiroaki-iwase** [comment]
I merged it.
So I close this pull request.

**Pull request merged**